### DISCUSSION TITLE
Armazenando o token

### DISCUSSION THREAD

**Discussion by AleDinizRJ** [balta-io/7187#41] [Perguntas e Respostas] [answered]
Prezados, boa noite!
Preciso de uma ajuda. Na aula sobre armazenar o token, eu implementei o código conforme o Balta ensinou. Na hora de realizar o login como teste e verificar no developer tools, o sistema gerou um erro, conforme mostra a imagem abaixo:

![image](https://user-images.githubusercontent.com/88723782/132079954-cb2b0bcd-b3b9-410f-b934-305ff3f60fb2.png)

Segue o código implementado:

1- login-page.component.ts
![image](https://user-images.githubusercontent.com/88723782/132079989-0799e4b9-3733-4a1e-8919-4e63605c27a5.png)

2- Data.services.ts
![image](https://user-images.githubusercontent.com/88723782/132080019-25053f91-bf34-492e-9bb5-8a38feeb0abd.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @AleDinizRJ este erro informa que sua API está fora... Consegue confirmar que ela está rodando, se sim, está sendo executada na porta 3000 mesmo?

**Comment by AleDinizRJ** [comment]
Olá Balta, bom dia!

Essa é uma grande dúvida que tenho ao assistir às aulas deste curso. Porque
eu estou seguindo a Carreira dos cursos de Angular. A única API feita nesta
carreira de cursos foi a TODO. Eu fico realmente confuso quando você se
refere a API destinada a este curso. Provavelmente o erro está ligado a API
que não está  rodando.

Favor me informar qual a API eu devo executar?
Ou melhor, se for o caso, qual ocurso eu tenho que fazer antes?

Att,

Alessandro Diniz.

Em seg., 6 de set. de 2021 às 07:28, Andre Baltieri <
***@***.***> escreveu:

> Bom dia @AleDinizRJ <https://github.com/AleDinizRJ> este erro informa que
> sua API está fora... Consegue confirmar que ela está rodando, se sim, está
> sendo executada na porta 3000 mesmo?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/balta-io/7187/discussions/41#discussioncomment-1285595>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVE5CRR4C2AY7HMXPQKACB3UASJULANCNFSM5DMZSGEQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Atc,

Alessandro Diniz

**Comment by AleDinizRJ** [comment]
Balta, boa noite!

Eu comecei a reler algumas coisas e vi que a API do curso 7187 está no curso 7180. 
Bom, eu não fiz o curso 7180. Por não estar na carreira dos cursos de Angular. 

Existe alguma maneira de executar essa aplicação sem fazer o curso 7180?

**Comment by AleDinizRJ** [comment]
Balta, boa noite!

Eu entendi o que houve. 
No vídeo que você explica sobre configurar o Backend eu segui as instruções para instalar o Mockoon, mas na hora de importar o arquivo , você não citou o nome do arquivo JSON e eu peguei o primeiro arquivo Json com o nome Mockoon. Portanto, esse arquivo não levantava a aplicação. Eu corrigi e importei o arquivo correto. Agora eu consigo gerar o token e utilizar o Postman.

Obrigado e desculpe qualquer coisa.

Mais uma situação. A tela de produtos da minha aplicação web, não está carregando as informações dos produtos como : preço, descrição e a imagem. Eu até pensei que Mockando a aplicação correta iria carregar as informações dos produtos, mas não foi.

Me ajuda em mais essa aí por gentileza.

**Reply by andrebaltieri** [reply]
Perfeito <3
